types/xelib/index.d.ts

```diff
@@ -0,0 +1,1629 @@
+// Type definitions for non-npm package xelib 0.6
+// Project: https://github.com/z-edit/xelib
+// Definitions by: Alex Layton <https://github.com/awlayton>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.4
+
+/**
+ * xelib instance
+ */
+export const wrapper: XELib;
+
+// Distinguish handles from regular numbers
+export type Handle = number & { __xelib_handle__: never };
```

Comment:
is that required? The code from README would have to be update to:
```ts
xelib.GetElement(0 as Handle, 'Skyrim.esm\\00012E46');
```
up to you folks, though.

Reply:
The library is actually a wrapper around a dll. The handles are only meaningful to the underlying code and a user of the library should not be passing anything that was not received from the library as a Handle.

There are cases where functions (e.g., `GetElement`) take `0` in place of a Handle. I tried to make all such functions accept `0` as well as `Handle` but it appears I missed that one.

I will get rid of that and just alias `Handle` to `number` if that is preferred.

Reply:
cool, that's perfectly fine with me. I've just pointed that in the readme-driven-development that nuance arise with that specific detail. 👍